baselines/asset/samples/generated/v1/asset_service.export_assets.js.baseline

```diff
@@ -0,0 +1,70 @@
+
+'use strict';
+
+function main() {
+  // [START asset_export_assets_sample]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  /**
+   *  Required. The relative name of the root asset. This can only be an
+   *  organization number (such as "organizations/123"), a project ID (such as
+   *  "projects/my-project-id"), or a project number (such as "projects/12345"),
+   *  or a folder number (such as "folders/123").
+   */
+  // const parent = 'abc123'
+  /**
+   *  Timestamp to take an asset snapshot. This can only be set to a timestamp
+   *  between the current time and the current time minus 35 days (inclusive).
+   *  If not specified, the current time will be used. Due to delays in resource
+   *  data collection and indexing, there is a volatile window during which
+   *  running the same query may get different results.
+   */
+  // const readTime = ''
+  /**
+   *  A list of asset types of which to take a snapshot for. For example:
+   *  "compute.googleapis.com/Disk". If specified, only matching assets will be
+   *  returned. See [Introduction to Cloud Asset
+   *  Inventory](https://cloud.google.com/asset-inventory/docs/overview)
+   *  for all supported asset types.
+   */
+  // const assetTypes = 'abc123'
+  /**
+   *  Asset content type. If not specified, no content but the asset name will be
+   *  returned.
+   */
+  // const contentType = ''
```

Comment:
What's the different between `contentType` and above `[contentType]`? (https://github.com/googleapis/gapic-generator-typescript/pull/952/files#diff-c13c1fba403a2b60dd3428eeb41510b54ce8d78c7b3fec53469e009790847194R30)

Why do we have `[]`?

Reply:
I think this is an issue with the API's designation of the variable. I added some special clauses to remove it.